.changeset/metal-plants-ring.md

```diff
@@ -0,0 +1,6 @@
+---
+"@marigold/docs": patch
+"@marigold/components": patch
+---
+
+refa([DST-978]): Add item and section as child component of `ActionMenu`
```

Comment:
Wouldn't this be a breaking Change?

Reply:
Why? :-O I just added stuff, They way it worked before still works. I think?

Reply:
You think or you know?  I thought because of ActionMenu.Item

Reply:
I know :P